src/components/Modal/index.js

```diff
@@ -32,12 +32,18 @@ export default class Modal extends PureComponent {
     ]),
     appendToBody: PropTypes.bool,
     onClose: PropTypes.func,
+    allowFullscreen: PropTypes.bool,
```

Comment:
should be the inverse, opting out should look like `<Modal disableFullscreen />` instead of `<Modal allowFullscreen={false} />`

Reply:
done